assets/src/scripts/components/hydrograph/axes.js

```diff
@@ -10,12 +10,33 @@ const { layoutSelector } = require('./layout');
 const { xScaleSelector, yScaleSelector } = require('./scales');
 const { yLabelSelector } = require('./timeSeries');
 
+const { USWDS_LARGE_SCREEN } = require('../../config');
 const { getCurrentDateRange, getCurrentParmCd } = require('../../selectors/timeSeriesSelector');
+const { mediaQuery } = require('../../utils');
 
-const INTERVAL = {
-    P7D: timeDay,
-    P30D: timeWeek,
-    P1Y: timeMonth
+const dateFormatter = timeFormat('%b %d');
+
+const FORMAT = {
+    P7D: dateFormatter,
+    P30D: dateFormatter,
+    P1Y: timeFormat('%b %Y')
+};
+
+const tickInterval = function(period) {
+    switch (period) {
+        case 'P7D':
+            return timeDay;
+        case 'P30D':
+            return timeWeek;
+        case 'P1Y':
+            if (mediaQuery(USWDS_LARGE_SCREEN)) {
```

Comment:
As an aside, looking into the image server I realized we'll have a challenge with these JS media queries, especially when SVG is the output format as it doesn't have a fixed size. Something to think about...